app/views/content_items/detailed_guide.html.erb

```diff
@@ -20,10 +20,10 @@
   <div class="column-two-thirds">
     <%= render 'govuk_component/metadata',
         from: @content_item.from,
+        part_of: @content_item.part_of,
         other: {
```

Comment:
when https://github.com/alphagov/static/pull/774 will be merged then this could become:
```
<%= render 'govuk_component/metadata',
        from: @content_item.from,
        part_of: @content_item.part_of,
        first_published: @content_item.published,
        last_updated: @content_item.updated,
        see_updates_link: true
%>
```
right?